### PR TITLE
Update to Minitest 5

### PR DESCRIPTION
**Pull request by nateberkopec**
+change namespace from MiniTest to Minitest

### CODE REVIEW

File: minitest-perf.gemspec [hunk 1]
```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.3"
   spec.add_development_dependency "rake"
-  spec.add_development_dependency "mocha"
-  spec.add_dependency "minitest", "~> 4.0"
+  spec.add_development_dependency "mocha", "~> 1.1"
+  spec.add_dependency "minitest", "~> 5"
```
Comment (zenspider) [inline minitest-perf.gemspec]: This should be `~> 5.0`.

Comment (nateberkopec) [inline minitest-perf.gemspec]: @zenspider Thanks! Fixed.

**Comment by nateberkopec** [comment]
I've removed `debugger` from the Gemfile, as it makes it impossible to develop this gem on Ruby 2.x.

File: Gemfile [hunk 2]
```diff
@@ -4,6 +4,6 @@ source 'https://rubygems.org'
 gemspec
 
 
-gem "debugger", platforms: :mri
+#gem "debugger", platforms: :mri
```
Comment (mrsimo) [inline Gemfile]: Would you mind deleting the line entirely?

Comment (nateberkopec) [inline Gemfile]: Done!

**Comment by mrsimo** [comment]
Give me a few minutes and I'll release a new version.

I saw the original PR is from November :( sorry it took me so long!

**Comment by mrsimo** [comment]
@nateberkopec new version is out https://rubygems.org/gems/minitest-perf

**Pull request merged**